spec/integration/data_stream/manifest.spec.yml

```diff
@@ -169,6 +169,31 @@ spec:
             hidden:
               description: Makes the data stream hidden
               type: boolean
+    route:
+      description: Routing rule definition
+      type: object
+      properties:
+        dataset:
+          type: string
```

Comment:
If this corresponds to `dataset` in the reroute processor, this can be also a list: https://www.elastic.co/guide/en/elasticsearch/reference/current/reroute-processor.html

Reply:
Updated.
Reading that definition, I set both string and array of strings for both `dataset` and `namespace` keys.